src/main/java/at/hannibal2/skyhanni/features/minion/InfernoMinionFeatures.kt

```diff
@@ -0,0 +1,77 @@
+package at.hannibal2.skyhanni.features.minion
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryOpenEvent
+import at.hannibal2.skyhanni.utils.KeyboardManager
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.nbt.NBTTagString
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class InfernoMinionFeatures {
+    private val config get() = SkyHanniMod.feature.minions
+    private val infernoMinionTitlePattern by RepoPattern.pattern(
+        "minion.infernominiontitle",
+        "Inferno Minion .*"
+    )
+    private val fuelItemIds = listOf(
```

Comment:
Move this list of minion fule items into the repo. This way you can directly load them as neu internal names